src/neo4j_graphrag/experimental/pipeline/pipeline.py

```diff
@@ -182,40 +189,100 @@ def show_as_dict(self) -> dict[str, Any]:
         return pipeline_config.model_dump()
 
     def draw(
-        self, path: str, layout: str = "dot", hide_unused_outputs: bool = True
+        self, path: str, layout: str = "force", hide_unused_outputs: bool = True
     ) -> Any:
-        G = self.get_pygraphviz_graph(hide_unused_outputs)
-        G.layout(layout)
-        G.draw(path)
+        """Draw the pipeline graph using neo4j-viz.
 
-    def get_pygraphviz_graph(self, hide_unused_outputs: bool = True) -> pgv.AGraph:
```

Comment:
I don't remember why we haven't made this method private, but since we are removing it we should mention it in the changelog.